x-pack/plugins/ingest_pipelines/public/application/sections/manage_processors/constants.ts

```diff
@@ -0,0 +1,176 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import type { GeoipDatabase } from '../../../../common/types';
+
+export const ADD_DATABASE_MODAL_TITLE_ID = 'manageProcessorsAddGeoipDatabase';
+export const ADD_DATABASE_MODAL_FORM_ID = 'manageProcessorsAddGeoipDatabaseForm';
+export const DATABASE_TYPE_OPTIONS = [
+  {
+    value: 'maxmind',
+    text: i18n.translate('xpack.ingestPipelines.manageProcessors.geoip.maxmindDatabaseType', {
+      defaultMessage: 'MaxMind',
+    }),
+  },
+  {
+    value: 'ipinfo',
+    text: i18n.translate('xpack.ingestPipelines.manageProcessors.geoip.ipinfoDatabaseType', {
+      defaultMessage: 'IPInfo',
+    }),
+  },
+];
+export const GEOIP_NAME_OPTIONS = [
+  {
+    value: 'GeoIP2-Anonymous-IP',
+    text: i18n.translate('xpack.ingestPipelines.manageProcessors.geoip.anonymousIPDatabaseName', {
+      defaultMessage: 'GeoIP2 Anonymous IP',
+    }),
+  },
+  {
+    value: 'GeoIP2-City',
+    text: i18n.translate('xpack.ingestPipelines.manageProcessors.geoip.cityDatabaseName', {
+      defaultMessage: 'GeoIP2 City',
+    }),
+  },
+  {
+    value: 'GeoIP2-Connection-Type',
+    text: i18n.translate(
+      'xpack.ingestPipelines.manageProcessors.geoip.connectionTypeDatabaseName',
+      {
+        defaultMessage: 'GeoIP2 Connection Type',
+      }
+    ),
+  },
+  {
+    value: 'GeoIP2-Country',
+    text: i18n.translate('xpack.ingestPipelines.manageProcessors.geoip.countryDatabaseName', {
+      defaultMessage: 'GeoIP2 Country',
+    }),
+  },
+  {
+    value: 'GeoIP2-Domain',
+    text: i18n.translate('xpack.ingestPipelines.manageProcessors.geoip.domainDatabaseName', {
+      defaultMessage: 'GeoIP2 Domain',
+    }),
+  },
+  {
+    value: 'GeoIP2-Enterprise',
+    text: i18n.translate('xpack.ingestPipelines.manageProcessors.geoip.enterpriseDatabaseName', {
+      defaultMessage: 'GeoIP2 Enterprise',
+    }),
+  },
+  {
+    value: 'GeoIP2-ISP',
+    text: i18n.translate('xpack.ingestPipelines.manageProcessors.geoip.ispDatabaseName', {
+      defaultMessage: 'GeoIP2 ISP',
+    }),
+  },
+];
+export const IPINFO_NAME_OPTIONS = [
+  {
+    value: 'asn',
+    text: i18n.translate('xpack.ingestPipelines.manageProcessors.ipinfo.freeAsnDatabaseName', {
+      defaultMessage: 'Free IP to ASN ',
```

Comment:
The most miniscule of nits: there's an extra space after "ASN" inside the string.